test/features/configure_bugs.feature

```diff
@@ -15,7 +15,7 @@ Feature: Functional test for configure sub level
     And     Online nodes are "hanode1 hanode2"
 
     # mask password by default
-    When    Run "crm node utilization hanode1 set password qwertyui" on "hanode1"
+    When    Run "crm node utilization hanode1 set password=qwertyui" on "hanode1"
```

Comment:
Instead of change the current test case, which is our preference conforms to the help text, I prefer to add one more test case. In the end, both are tested.

Reply:
Changed